examples/Console/TestOtlpExporter.cs

```diff
@@ -74,7 +74,12 @@ private static object RunWithActivitySource(string endpoint, string protocol)
                     .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("otlp-test"))
                     .AddOtlpExporter(opt =>
                     {
-                        opt.Endpoint = new Uri(endpoint);
+                        // If endpoint was not specified, the proper one will be selected according to the protocol.
```

Comment:
it'd be good to print out the entire url and protocol that will be used by this app, when it runs. Something like https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/TestPrometheusExporter.cs#L86